### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Added login, signup page, cart and wishlist page

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -123,7 +123,9 @@ section {
   .home .slide .content h3 {
     margin-top: 15rem;
   }
-  .form-box, .cart-grid {
+  .form-box 
+  /* .cart-grid */
+   {
```
Comment (harshitpaliwal95) [inline src/App.css:128]: remove unused css

File: src/pages/Cart/Cart.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                        Stylish Bags
+                      </h3>
+      
+                      <div className="sm-gap md-gap">
+                        <span className="txt-bold"> Rs.3899</span>
+                        <span className="txt-crossed-off">Rs.2999</span>
+                        <span className="txt-high-light">10% Off</span>
+                      </div>
+                      
+                      <div className="txt-gray">
```
Comment (harshitpaliwal95) [inline src/pages/Cart/Cart.js:36]: can create cart component to reduce extra lines of code

File: src/pages/Wishlist/Wishlist.js [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+                  </span>
+                </h3>
+
+                <div className="sm-gap">
+                  <span className="txt-bold"> Rs.399</span>
+                  <span className="txt-crossed-off">Rs.599</span>
+                  <span className="txt-high-light">10% Off</span>
+                </div>
+
+                <div class="card-footer">
```
Comment (harshitpaliwal95) [inline src/pages/Wishlist/Wishlist.js:83]: use components

**Review by harshitpaliwal95 — COMMENTED** [review]
- remove commented code before merging the PR
-  create component to reduce extra line of code
 overall looks good and nice ui

File: src/pages/Auth/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+export function SignUp() {
+  return (
+    <div>
+        <div className="form-box">
+      <div className="button-box">
+        <h3 className="toggle-btn">SIGN UP</h3>
+      </div>
+      <form className="input-group">
+      <label className='password-label'>First Name</label>
```
Comment (arpit01923) [inline src/pages/Auth/SignUp.js:13]: use of components here should make your code clean and readable

File: src/pages/Cart/Cart.js [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+export function Cart() {
+  return (
+    <div>
+    <h3 className="cart-heading">My Cart <span className="cart-desc">(5)</span></h3>
+    <h4 className="empty-list">Your Cart Is Empty ! ☹️</h4>
+    <div className="cart-grid">
+    <main className="main-cart">
+        <ul className="cart-products">           
+
+            <li className="cart-main">
```
Comment (arpit01923) [inline src/pages/Cart/Cart.js:15]: components should be used here to make a clean code

**Review by arpit01923 — COMMENTED** [review]
-left some comments, resolve them
-make your code clean and readable by using components and with the hint (map)

**Pull request merged**